src/Parsers/queryNormalization.h

```diff
@@ -61,31 +62,39 @@ inline UInt64 ALWAYS_INLINE normalizedQueryHash(const char * begin, const char *
             /// By the way, there is padding in columns and pointer dereference is Ok.
             || (token.type == TokenType::BareWord && *token.end != '('))
         {
-            /// Identifier is complex if it contains whitespace or more than two digits
-            /// or it's at least 36 bytes long (UUID for example).
-            size_t num_digits = 0;
-
-            const char * pos = token.begin;
-            if (token.size() < 36)
+            /// Explicitly ask to normalize with identifier names
```

Comment:
It probably should be "... to normalize keeping/retaining identifier names".

Reply:
Done